cobalt/media/base/metrics_provider.cc

```diff
@@ -151,6 +151,75 @@ void MediaMetricsProvider::ReportActionLatencyUMA(
       UMA_HISTOGRAM_TIMES("Cobalt.Media.SbPlayer.Destroy.LatencyTiming",
                           action_duration);
       break;
+<<<<<<< HEAD
+=======
+    case MediaAction::SBPLAYER_GET_PREFERRED_OUTPUT_MODE:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbPlayer.GetPreferredOutputMode.LatencyTiming",
+          action_duration, base::TimeDelta::FromMicroseconds(1),
+          base::TimeDelta::FromMicroseconds(100), 50);
```

Comment:
Changed this to `base::TimeDelta::FromMilliseconds(5)` as this is a safe range for the histogram.